cloudformation/template.yaml

```diff
@@ -759,6 +886,9 @@ Resources:
           WEBSITE_BUCKET_NAME: !Ref DevPortalSiteS3BucketName
           StaticBucketName: !Ref ArtifactsS3BucketName
           CustomersTableName: !Ref DevPortalCustomersTableName
+          FeedbackTableName: !Ref DevPortalFeedbackTableName
+          FeedbackSnsTopicArn:
+            Fn::If: [EnableFeedbackSubmission, !Ref FeedbackSubmittedSNSTopic, '']
```

Comment:
Doesn't matter in the slightest, but "Fn::If" vs "!If"?

Reply:
Nice catch - updated. I think we'll want to continue with the `!` syntax as we aren't using `Fn::` anywhere else in the template